### ISSUE TITLE
Exclamation Marks in Reporting section, No reports can be generated

### ISSUE THREAD

**Issue by PRijnbeek** [OHDSI/Atlas#402]
What do the black exclamation marks mean in the reporting section of Atlas? I could not find this in the documentation.

I guess this means the report is not created yet?

When I select some reports and click on Generate nothing happens, also no error when using the development tools in Chrome? Any ideas?

Thanks

**Comment by fdefalco** [comment]
Let me start by saying that this whole module is being rewritten and I apologize for its current state which is largely my fault. 

There are actually two different black exclamation mark states - yes, its that bad...

In some cases you will see a black _circle_ background with a white exclamation mark on it which signifies that the cohort has not been generated for that data source.

In other cases you will see a black _triangle_ background with a white exclamation mark along with a checkbox next to it.  In this case the cohort has been generated but the report has not been generated.  Here you can check the check box and then click the generate button below (which provides **no feedback** when clicked and often leads to people unknowingly kicking off a bunch of jobs not realizing it worked the first time they clicked.

As I said, all I can do is apologize and weep tears of sadness that this is still in this state.

**Comment by PRijnbeek** [comment]
Aah thanks Frank no worries I understand the load of work..

I indeed see a Hercules Job running and then Completed but I do not see a finished report. Maybe this has to do with a problem with the Hercules tables in the result schema..? We will check.

By the way what we have built (now outside of Atlas) is an import functionality that reads the Export of the Reporting tab back into the result schema of a central Atlas install (after selecting database name). Would be nice to have a chat with you about this option as well. This allows us to do this:


1.       Define Cohort in Atlas

2.       Export Cohort as JSON

3.       Distribute it to partners using a task management system

4.       Data Custodian upload the results to a Remote Research Environment

5.       Import functionality on the central Atlas

6.       Visualization of all the reports of all the databases

If things will change in Hercules it would be good to know.

Peter
